aws/main.tf

```diff
@@ -279,13 +279,43 @@ chmod 600 /home/${var.default_ssh_user}/.ssh/authorized_keys
 
 # Install NFS utilities and mount the EFS file system at /opt/thirdai_platform/model_bazaar
 
+sleep 60
```

Comment:
When the user_data scripts runs at the start, the network and those things were not yet ready and apt update was failing because it cannot pull things, so community suggested that wait for 30-60sec for machine to get setup with network and stuff.